CMakeLists.txt

```diff
@@ -102,22 +117,28 @@ macro(jerry_add_link_flags)
 endmacro()
 
 # build mode specific compile/link flags
+if(USE_GCC)
 set(CMAKE_C_FLAGS_RELEASE "-Os")
+endif()
 
 # Architecture-specific compile/link flags
 jerry_add_compile_flags(${FLAGS_COMMON_ARCH})
 jerry_add_flags(CMAKE_EXE_LINKER_FLAGS ${FLAGS_COMMON_ARCH})
 
 # Enable static build
 if(ENABLE_STATIC_LINK)
-  jerry_add_link_flags("-static")
+  if (USING_GCC_OR_CLANG)
```

Comment:
I see :). If the ENABLE STATIC_LINK is off, please set its value to ON and write a STATUS message that the ENABLE_STATIC_LINK has been enabled.